emperor/support_files/js/util.js

```diff
@@ -89,6 +89,27 @@ define(['underscore'], function(_) {
 
   /**
    *
+   * Split list of string values into numeric and non-numeric values
+   * @param {String[]} values The values to check
+   * @return {Numeric[]} Array of numeric values, converted to numbers
```

Comment:
Shouldn't this be a list of lists? Or is the convention to use two `@return` tags?